### PR TITLE
Dev/jgough/v2 schema

### PR DESCRIPTION
**Pull request by honourfish**
PR for creating a v2 schema that doesn't force us to use permissioned identities for the canonical simple hash anchoring schema.

Fixes AB#7958

### CODE REVIEW

**Review by eccles — CHANGES_REQUESTED** [review]
So v1.py has been modified to remove 'public' where appropriate and v2.py does not bother (i.e. its the same as the original v1.py). Is that correct and expected.

Also are there unittests and has the README.md been modified to mention v2 as well as v1?

**Comment by honourfish** [comment]
> So v1.py has been modified to remove 'public' where appropriate and v2.py does not bother (i.e. its the same as the original v1.py). Is that correct and expected.
> 
> Also are there unittests and has the README.md been modified to mention v2 as well as v1?

Yes! thats exactly it, have updated the unittests, will also update the readme!

**Comment by honourfish** [comment]
updated readme

File: README.rst [hunk 1]
```diff
@@ -201,3 +318,31 @@ Querying the public assets (does not require authentication)
...
+    endpoint = "archivist/v2/publicassets/-/events"
+    
+    api_query="https://app.rkvst.io"
+    api_query+="/archivist/v2/publicassets/-/events"
+    api_query+="?proof_mechanism=SIMPLE_HASH"
+    api_query+="&timestamp_accepted_since=2022-10-07T07:01:34Z"
+    api_query+="&timestamp_accepted_before=2022-10-16T13:14:56Z"
+    api_query+="&order_by=SIMPLEHASHV1"
+
+    CLIENT_ID=$(cat credentials/client_id)
```
Comment (eccles) [inline README.rst]: is this needed?

**Review by eccles — CHANGES_REQUESTED** [review]
LGTM

When will the user use one of the 4 options presented in the README. maybe a bit of text at the top of READMEto point the user to teh correct option?

File: README.rst (hunk 1 above)
Comment (honourfish) [reply README.rst]: not for public no

**Review by eccles — APPROVED** [review]

**Pull request merged**